script.js

```diff
@@ -1,18 +1,36 @@
+// create variables
 var input;
 var items = [];
 var count = 0;
 
+// while the input is not blank, prompt user to enter an item
 while(input != ""){
 	input = prompt("Enter item");
-
-	
+// this appends the input into items as a list
+	items.push(input);
 }
 
+// remove the last item from the array
+items.pop();
 
-count = 0;
+// create variables
 var toDisplay = "";
+var numbered;
+var numberedList = [];
+
+// while count is less than the item length,
 while(count < items.length){
-	
-	
+//append number (by adding 1 to the count) and (eg 1. ) to the items and save it as the variable numbered
+	numbered =  count+1 + "." + items[count] + "\n";
```

Comment:
This is where you're building your string. Change this line to the following
```
toDisplay = toDisplay + (count+1)+". "+items[count]+"\n";
```
You're overcomplicating this one. If you build `toDisplay` just like that, all you need to do is ..well... display it. No need for `numberedList` or `numbered`